### PR TITLE
ET-686 ETP's AI fields on classic editor

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-686]

Relates to https://github.com/moderntribe/event-tickets-plus/pull/956

Artifacts (also shared to Testing Instructions):

- 2min video that things are working as expected with ETP required Attendee Info: https://share.getcloudapp.com/JruWlxkb
- 1min video showing things still work even when ETP is inactive: https://share.getcloudapp.com/mXuqZKdW

[ET-686]: https://moderntribe.atlassian.net/browse/ET-686

### CODE REVIEW

File: src/resources/js/rsvp.js [hunk 1]
```diff
@@ -20,65 +20,91 @@ var tribe_tickets_rsvp = {
...
 		} );
 
-		if (
-			0 === rsvp_qty ||
-			! $.trim( $rsvp.val() ).length ||
-			! $.trim( $name.val() ).length ||
-			! $.trim( $email.val() ).length
-		) {
-			return false;
+		return !! (
```
Comment (sc0ttkclark) [inline src/resources/js/rsvp.js]: Why !! here?

File: src/resources/js/rsvp.js [hunk 2]
```diff
@@ -20,65 +20,91 @@ var tribe_tickets_rsvp = {
...
+		const $form = $( this ).closest( 'form' );
+
+		const $rsvpMessages = $form.find(
+			'.tribe-rsvp-messages, ' +
+			'.tribe-rsvp-message-confirmation-error',
+		);
+
+		const $etpMetaMessages = $form.find( '.tribe-event-tickets-meta-required-message' );
+
+		const isRsvpInfoValid = !! my.validate_rsvp_info( $form );
```
Comment (sc0ttkclark) [inline src/resources/js/rsvp.js]: Why !! here?

File: src/resources/js/rsvp.js [hunk 3]
```diff
@@ -20,65 +20,91 @@ var tribe_tickets_rsvp = {
...
+		const isAttendeeMetaValid = !! my.validate_meta( $form );
```
Comment (sc0ttkclark) [inline src/resources/js/rsvp.js]: Why !! here?

**Comment by cliffordp** [comment]
@sc0ttkclark Good catches. I was just forcing to boolean `!!` where I knew I expected it, not first looking if forcing was necessary. Removed now, tested still working locally, and PR updated.

**Pull request merged**